### PR TITLE
Fixing unused weight_decay argument

### PR DESCRIPTION
**Pull request by ethanjperez**
Currently the L2 regularization is hard-coded to "0.01", even though there is a --weight_decay flag implemented (that is unused). I'm making this flag control the weight decay used for fine-tuning in this script.

### CODE REVIEW

**Comment by thomwolf** [comment]
Indeed, thanks Ethan!
